contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -177,7 +175,7 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         // will be cleared from state. At this point, you can not add to its fee. This is also the
         // case if the given message never existed.
         require(
-            sentMessageInfo[destinationBlockchainID][messageID].messageHash != bytes32(0),
+            sentMessageInfo[messageID].messageHash != bytes32(0),
```

Comment:
i think we looked at this before for gas savings, and not part of PR, but surprised we do so many look up of a state variable, instead of saving in local variable.